workspaces/rbac/plugins/rbac/src/components/CreateRole/PermissionPoliciesFormNestedRow.tsx

```diff
@@ -148,7 +148,8 @@ const PermissionPoliciesFormNestedRow = ({
                 disabled={
                   permissionPolicyRowIndex < 0 ||
                   !permissionPolicy.isResourced ||
-                  !conditionRulesLength
+                  !conditionRulesLength ||
+                  permissionPolicy.permission === 'policy.entity.create'
```

Comment:
@PatAKnight we shouldnt hard code this for one permission policy as this similar case might occur in future for some other plugin. Can we remove resource-type from `policy.entity.create` as it will not be used for adding conditons anyways here. LMK if we have a better way of doing it or this is the only way for now.